boot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
with this implementation there is absolutely no need to have a boot script, however almost every project I put on docker eventually starts needing more runtime commands than can fit in one CMD in Dockerfile, so im just getting it out of the way and booting from this now

Reply:
Wouldn't it be doable and better to put this stuff in the docker file?